plugins/duck.yaml

```diff
@@ -19,7 +20,6 @@ spec:
     sha256: 86294517e15b77297eeeafcb515ed7dd5f1c0ea13f6bfa4e1ecdc9d8096ab009
     files:
     - from: "*"
```

Comment:
you need to remove both from/to to take advantage of the default.

Reply:
sounds good! I am going off what kubectl-tree is doing.